kwave/options.py

```diff
@@ -1,6 +1,7 @@
 from dataclasses import dataclass
 import numpy as np
 from kwave.utils import get_h5_literals
+from kwave.utils import get_optimal_pml_size
```

Comment:
How did it work before the change without this import_

Reply:
neither do I...